packages/component/src/Utils/addTargetBlankToHyperlinksMarkdown.spec.js

```diff
@@ -1,7 +1,12 @@
+// TODO: [P4] Object.fromEntries is not on Node.js 11.*
+//       If all devs are on Node.js >= 12.0, we can remove "core-js"
```

Comment:
WC devs?

Reply:
Do we need to leave a note in the `CONTRIBUTING.md`?

Reply:
Will move to `CONTRIBUTING.md`.